src/main/java/org/variantsync/diffdetective/diff/difftree/serialize/nodeformat/MappingsDiffNodeFormat.java

```diff
@@ -3,15 +3,15 @@
 import org.variantsync.diffdetective.diff.difftree.DiffNode;
 
 /**
- * Print CodeType and DiffType and Mappings of Macros.
- * The produced label will be <code>CodeType_DiffType_"macro formula"</code> for mapping nodes,
- * and <code>CodeType_DiffType_""</code> for non-mapping nodes.
- * @see DiffNodeLabelPrettyfier#prettyPrintIfMacroOr(DiffNode, String)
+ * Print NodeType and DiffType and Mappings of Annotations.
+ * The produced label will be <code>NodeType"annotation formula"</code> for mapping nodes,
+ * and <code>NodeType""</code> for non-mapping nodes.
```

Comment:
This comment is lying now. While renaming `CodeType` to `NodeType` I seemingly got this comment wrong.